README.md

```diff
@@ -5,110 +5,47 @@
-
-**Create Database**
-
-By typing the command ```python reset-db.py``` a database file containing the correct schemas will be created in the data directory with the name ```advancement.sqlite```.
-
-**Populate Database**
-
-The ```reset-db.py``` will only create empty tables for you, in order to populate the database you will need to execute the command: ```python add_dummy.py```. This file will add dummy data to the system so that you can gauge how the system is supposed to run.
-
-**How to View the Database** 
-
-Now that you have the database created and populated with data, you are probably asking yourself how do I see that? Our system development team likes to use a tool called [DB Browser](http://sqlitebrowser.org/). This tool is a visual way of viewing and editing SQLite database files. 
-
-![dbBrowser.PNG](https://bitbucket.org/repo/bEXb4L/images/3023751797-dbBrowser.PNG)
-
-**Step Three: Running the Application**
-
-The only remaining step to getting your development environment deployed is running the actual application. This can be achieved through the command ```python run.py```, when you run this command you should see a URL created for you. 
-
-![run.PNG](https://bitbucket.org/repo/bEXb4L/images/1543001500-run.PNG)
-
-The URL will take you to the application and allow you to see any changes you make to the system. That's all that has to be done in order to get the development environment created and ready for editing.
-
-[TOC]
-#Installation#
+# BCSR Application Installation
 
 ##Requirements##
 * python 2.7
-* linux, unix, mac, windows(with attachments)
-* git
-
-## Creating Development Environment 
-
-1. ** Fork ** the repository from BitBucket and rename the project 
-
-2. If working on a **local machine**, then clone the repo from your terminal. 
-
-3. If you are working from cloud 9 follow [these steps](https://codymyers93.wordpress.com/2016/03/07/octoprint-working-in-cloud-9/) for directions on how to setup cloud 9 with git.
-
-4. After you have successfully clone the repo. Run:
-``` bash
-source setup.sh
-python app.py
-```
-
-If you are successful you will see something like:
-``` bash
+* linux, unix, mac, windows(with attachments), Ubuntu
+
+## Creating Development Environment
+1. Clone the BCSR repository (from Github) in your home directory: ```git clone <URL>```
+2. If working on a **local machine**, then clone the repo from your terminal.
+3. Edit your ```secret_config.yaml``` file by referencing the example_secret_config.yaml file.
+4. Run ```source setup.sh```
+
+## Creating the database
+### PHPMYADMIN
+1. Go to http://0.0.0.0/phpmyadmin/ replacing the 0.0.0.0 with your server
+2. Log in with the credentials used in secret_config.yaml
+3. Click on "New" on top of the sidebar at the left of the screen
+4. If there is already a database named "bcsr", drop it
+5. Create a new database with the name "bcsr"
+6. Run ```python create_db.py```
```

Comment:
I get this error when trying to create databases

```
(venv) ramsayb2@Berea-C02ZC049LVDN:~/dev/bcsr %>python create_db.py                                 0 - ttys004
Traceback (most recent call last):
  File "/Users/ramsayb2/dev/bcsr/create_db.py", line 14, in <module>
    sqlite_dbs  = [ conf['databases']['dev']
KeyError: 'databases'
```

Reply:
There are codes that we edited and this line was changed to `sqlite_dbs  = [ conf['databases'][0] `. Is it maybe you haven't merged those codes in ur branch yet?